### PR TITLE
feat(web): UI mission detail

### LINKED ISSUE #122: [2JH] UI Mission Detail
## Describe the issue
C'est une feature, la page devrait montrer les détails d'une mission, cette page doit être liée en cliquant le bouton 'Open' de la page mission.

## Resource request
Trouver le design sur Figma

## Definition of done
- [x] Faire un sous-menu de cette page(SCOPE/RECON)
- [x] Ajout de la pagination
- [x] Ajout des badges(ReactJS, SQLi,XSS...etc...)
- [x] Ajout de la barre de Search
- [x] Faire la function de Search pour que la pagination montre directement le nom qu'on cherche
- [x] Connecter la page 'Add vulnerability' avec le bouton "ADD"

## JEH
2

### PR DESCRIPTION
**Pull request by claraxuxu**
# Description

Please provide a detailed description of what was done in this PR.
Precise the issue that you are resolving.

# Fixes

Specify which issue this PR targets, like so:

- fixes #122 

# Changes include

- [ ] Bugfix (non-breaking change that solves an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality)

# Breaking changes

Please complete this section if any breaking changes have been made, otherwise delete it.

# Checklist

- [x] I have assigned this PR to myself
- [ ] I have added @vrn-sh/members to the reviewers, or specific members of the team
- [x] I have added the needed labels
- [x] I have tested this code
- [ ] I have added / updated tests (unit / functionals / end-to-end / ...)
- [ ] I have updated the README and other relevant documents (guides...)

# Additional comments

Please post additional comments in this section if you have them, otherwise delete it.

### CODE REVIEW

**Review by djnnvx — APPROVED** [review]

File: web/src/config.ts
Comment (Ananane) [inline web/src/config.ts:1]: Ca marche si tu fais ça ?

File: web/src/config.ts [hunk 1]
```diff
@@ -1,9 +1,8 @@
 let apiUrl = '';
 
-if (import.meta.env.NODE_ENV === 'development')
-    apiUrl = import.meta.env.LOCAL_API_URL as string;
-else if (import.meta.env.NODE_ENV === 'production')
-    apiUrl = import.meta.env.REMOTE_API_URL as string;
+if (process.env.NODE_ENV === 'development') apiUrl = 'http://localhost:8080';
+else if (process.env.NODE_ENV === 'production')
+    apiUrl = 'https://voron.djnn.sh/api';
```
Comment (Ananane) [inline web/src/config.ts]: Ca marche si tu fais ça comme ça ?

Comment (claraxuxu) [reply web/src/config.ts]: yes ça marche comme ça, mais en vrai avec les variables est mieux, donc on est encore en train de voir la solution avant de merge

**Review by Ananane — APPROVED** [review]

File: web/src/config.ts (hunk 1 above)
Comment (djnnvx) [reply web/src/config.ts]: Cette solution ne permet pas a d'autre gens d'heberger ca, ou alors il faudrait qu'ils changent le front mais bon c'est un peu garbage, une autre solution est a preconniser effectivement mais pour le moment je sais pas comment faire

**Pull request merged**